templates/taxi/manufacturer_form.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>Create new manufacturer</h1>

```

Comment:
You will see the inscription "create" and during updating. Use `{{ object|yesno:"Update,Create" }}`

Reply:
done